library/src/doo/core.clj

```diff
@@ -238,9 +241,9 @@ where:
    (run-script js-env compiler-opts {}))
   ([js-env compiler-opts opts]
    {:pre [(valid-js-env? js-env)]}
-   (let [doo-opts (merge default-opts opts)
+   (let [doo-opts (merge default-opts {:base-dir (:output-to compiler-opts)} opts)
```

Comment:
What if `:base-dir` (perhaps renamed to `:working-dir` or `:exec-dir`) needed to be set explicitly? I think that could resolve the issues we're having in CI?